src/vitepress/components/VPNavBarSearch.vue

```diff
@@ -227,26 +228,14 @@ function load() {
   height: 22px;
   padding: 0;
   margin: 0;
-}
-
-.DocSearch-Button .DocSearch-Button-Key:first-child {
-  font-size: 1px;
-  letter-spacing: -1px;
-  color: transparent;
-}
-.DocSearch-Button .DocSearch-Button-Key:first-child::after {
-  content: v-bind(metaKey);
-}
-
-.DocSearch-Button .DocSearch-Button-Key:first-child::after,
-.DocSearch-Button .DocSearch-Button-Key:last-child {
-  font-size: 12px;
-  letter-spacing: normal;
   color: var(--vt-c-text-3);
   transition: color 0.5s;
 }
-.DocSearch-Button:hover .DocSearch-Button-Key:first-child::after,
-.DocSearch-Button:hover .DocSearch-Button-Key:last-child {
+.DocSearch-Button:hover .DocSearch-Button-Key {
   color: var(--vt-c-brand-light);
 }
+
+.DocSearch-Button[preload] .DocSearch-Button-Key:first-child::after {
+  content: v-bind(metaKey);
```

Comment:
I also found that the default `metaKey` value (`'Meta'`) will cause a layout shift. Any idea to fix this?

Reply:
I tried a much-simplified way. What do you think about it now?

Reply:
Update: I moved the button into a `<ClientOnly>` component. So it won't be pre-rendered in the server side. That also means the hotkey will only show up after we already know which OS it's on.